README.md

```diff
@@ -22,39 +22,39 @@
   
 ### Purpose Summary
 * C++ 17
-* Focus on native game dev (quick starting, easy to use, fast)
+* Focuses on native game dev (easy to use, fast deployment, intuitive)
 * Bugfixes ASAP
   
 ### Highlight Features
 * Windows x64 build support
-* Reimplement HttpClient based on yasio for process http request concorrently.
-* **More clearly third-party libs ['Upstream-Version-License'](thirdparty/README.md) for easy to publish your commercial apps based on adxe. Also some links to third party libs which supporting adxe too.**
+* Reimplement HttpClient based on yasio for concorrent http requests processing.
+* **clearer third-party libs ['Upstream-Version-License'](thirdparty/README.md) for easier publishing of your commercial apps based on adxe. Also some links to third party libs which support adxe too.**
 * Refactor AudioEngine, OpenAL for all platforms
-  * [OpenAL Soft](https://github.com/kcat/openal-soft), pass -DBUILD_DEP_ALSOFT=ON to cmake to force enable it
-  * [OpenAL.framework](https://opensource.apple.com/tarballs/OpenAL), if no ```BUILD_DEP_ALSOFT``` option specified, cmake script will choose it on osx/ios, even through it was mark as deprecated, but still available.
+  * [OpenAL Soft](https://github.com/kcat/openal-soft), pass -DBUILD_DEP_ALSOFT=ON to cmake to force enabling it
+  * [OpenAL.framework](https://opensource.apple.com/tarballs/OpenAL), if no ```BUILD_DEP_ALSOFT``` option specified, cmake script will choose it on osx/ios, even though it was marked as deprecated, but still available.
 * Refactor UserDefault with [mio](https://github.com/mandreyel/mio)
-* Modularize all optional extension, move from engine core to folder extensions
+* Modularize all optional extensions, move from engine core folder to an extensions folder
 * Implement all .wav formats supported by ```OpenAL Soft```, such as MS-ADPCM, ADPCM, ...
-* Use modern GL loader ```Glad```
+* Use a modern GL loader ```Glad```
 * Google [angle](https://github.com/google/angle) renderer backend support
 * C++ 17 standard
 * IOS SDK 9.0 as minimal deployment
 * Use fast pugixml
 * Use [curl](https://github.com/curl/curl) for transferring data with URL syntax
-* Use SAX parser for all plist file
+* Use SAX parser for all plist files
 * Spine-3.8 support
 * Extension ```FairyGUI``` support
-* ASTC 4x4/6x6/8x8 support (if hardware decoder not present, use software decoder)
-* ETC2 RGB/RGBA support    (if hardware decoder not present, use software decoder)
+* ASTC 4x4/6x6/8x8 support (if hardware decoding is not present, software decoding will be used instead)
```

Comment:
This isn't accurate, and the original is actually correct:

"decoding" is the action that the decoder will carry out. if one exists.  It's checking for a hardware decoder, and not the act of decoding.

So, `(if hardware decoder not present, use software decoder)` is spot on compared to `(if hardware decoding is not present, software decoding will be used instead)`.

Reply:
It's actually right if you look at it the right way, in english whenever you use ing it doesnt have to be 100% the action of doing something in the present.. somtimes it just refers to the thing itself or the action in general, I learned this somewhere but don't actually remember.

Reply:
I understand, and it makes sense.  

Regarding the original text, it is correct.  The statement is to indicate what action the software will take if a condition is not met.  That condition is the availability of a hardware decoder, or alternatively, the support for hardware decoding.

Reply:
So the right thing to say is 
```If hardware decoder is not present, software decoder will be used instead```
felt that the change is a bit wordy

Reply:
Oh I understand your concern, which is the the `use software decoder` part of the sentence.  

This also works and is perhaps less wordy: "if hardware decoding is not supported, then software decoding is used"

Reply:
Yeah this is cool